docs/source/installation.rst

```diff
@@ -1,2 +1,228 @@
+   | Option                        | Values                 | Default     | Description                                |
+   +===============================+========================+=============+============================================+
+   | ``BUILD_TYPE``                | ``Release``, ``Debug``,| ``Release`` | Build type used in the script              |
+   |                               | ``RelWithDebInfo``,    |             | by CMake for single-target generators      |
+   |                               | ``MinSizeRel``         |             | (e.g., Makefile Generators) and            |
+   |                               |                        |             | by a compiler for multi-target generators  |
+   |                               |                        |             | (e.g., Visual Studio Generators).          |
+   +-------------------------------+------------------------+-------------+--------------------------------------------+
+   | ``BUILD_SHARED_LIBS``         | ``ON``, ``OFF``        | ``OFF``     | Equivalent to the CMake YAGIT option.      |
+   +-------------------------------+------------------------+-------------+--------------------------------------------+
+   | ``INSTALL_DEPENDENCIES``      | ``OFF``, ``LOCAL``,    | ``LOCAL``   | Method of installing dependencies.         |
+   |                               | ``GLOBAL``, ``CONAN``  |             | Value ``OFF`` doesn't install any          |
+   |                               |                        |             | dependencies, and it assumes               |
+   |                               |                        |             | they are already installed system-wide.    |
+   |                               |                        |             | Value ``LOCAL`` installs dependencies      |
+   |                               |                        |             | locally in the *build/deps* directory.     |
+   |                               |                        |             | Value ``GLOBAL`` installs dependencies     |
+   |                               |                        |             | system-wide. It requires admin privileges. |
+   |                               |                        |             | Value ``CONAN`` installs dependencies      |
+   |                               |                        |             | using `Conan`_ package manager.            |
+   |                               |                        |             | It requires installed Conan. This option   |
+   |                               |                        |             | is only available in *setup.sh*.           |
+   +-------------------------------+------------------------+-------------+--------------------------------------------+
+   | ``GAMMA_VERSION``             | ``SEQUENTIAL``,        | ``THREADS`` | Equivalent to the CMake YAGIT option.      |
```

Comment:
Its unclear to me what `Equivalent to the CMake YAGIT option` means

Reply:
Maybe adding a comment, "see below" may help. On first reading I am not aware that CMake options will go below

Reply:
That's why I have added this line above the table
"Some of the options are the same as in the CMake YAGIT options section, as they are simply passed to CMake." with a link to that section.